### PR TITLE
Integration tests for mfa method form

### PR DESCRIPTION
**Pull request by arnav28**

### CODE REVIEW

File: ui/app/templates/components/mfa/method-form.hbs [hunk 1]
```diff
@@ -28,6 +29,7 @@
   @isActive={{this.isEditModalActive}}
   @confirmText={{@model.type}}
   @onConfirm={{perform this.save}}
+  @testSelector="mfa"
```
Comment (hashishaw) [inline ui/app/templates/components/mfa/method-form.hbs:32]: Is this necessary? We should instead follow our convention of using `data-test-*` selectors for test selectors

File: ui/tests/integration/components/mfa/method-form-test.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+  setupMirage(hooks);
+
+  hooks.beforeEach(function () {
+    this.store = this.owner.lookup('service:store');
+    this.model = this.store.createRecord('mfa-method');
+    this.model.type = 'totp';
+  });
+
+  test('it should render correct fields', async function (assert) {
+    await render(hbs`
```
Comment (hashishaw) [inline ui/tests/integration/components/mfa/method-form-test.js:18]: Can we put `assert.expect` on this test too please?

File: ui/app/templates/components/mfa/method-form.hbs (hunk 1 above)
Comment (arnav28) [reply ui/app/templates/components/mfa/method-form.hbs:32]: This Confirmation model component uses testSelector value and prefixes it with data-test-confirmation-modal.

File: ui/tests/integration/components/mfa/method-form-test.js (hunk 2 above)
Comment (arnav28) [reply ui/tests/integration/components/mfa/method-form-test.js:18]: Sure. 👍

File: ui/tests/integration/components/mfa/method-form-test.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+      <Mfa::MethodForm
+        @hasActions="true"
+        @model={{this.model}}
+      />
+      <div id="modal-wormhole"></div>
+    `);
+    assert.dom('[data-test-input="issuer"]').hasValue('Vault', 'Issuer input is populated');
+    assert.dom('[data-test-ttl-value="Period"]').hasValue('30', 'Period input ttl is populated');
+    let SHA512radioBtn = this.element.querySelectorAll('input[name=algorithm]')[2];
```
[the remark this answers is not in the document]
Comment (arnav28) [reply ui/tests/integration/components/mfa/method-form-test.js]: Oh true, thanks!!

**Pull request merged**